test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -307,6 +307,66 @@ TEST_F(HttpFilterTest, ErrorFailClose) {
   EXPECT_EQ(1U, config_->stats().error_.value());
 }
 
+// Test when failure_mode_allow is NOT set and the response from the authorization service is Error, but the runtime guard set to false
+// that the request is not allowed to continue and should forward error status code and body.
+TEST_F(HttpFilterTest, ErrorFailCloseWithRuntimeGuardFalse) {
```

Comment:
Let's test the new behavior first. Later we can create a helper function that has a boolean to set the runtime flag.

Reply:
Since we will not have any changes on `filters/http/ext_authz` we don't need to change anything on this test, right?